tests/driver/Firestore/Trigger/InProcessFirestore.onDelete.test.ts

```diff
@@ -0,0 +1,103 @@
+import { IFirebaseDriver, IFirestoreDocumentSnapshot, InProcessFirebaseDriver } from "../../../../src"
+
+describe("onDelete trigger of in-process Firestore", () => {
+    let driver: InProcessFirebaseDriver & IFirebaseDriver
+
+    beforeEach(() => {
+        driver = new InProcessFirebaseDriver()
+    })
+
+    test("onDelete handler is not triggered on doc create", async () => {
+        // Given we set up an onDelete handler on a collection;
+        const receivedSnapshots: IFirestoreDocumentSnapshot[] = []
+        driver
+            .runWith()
+            .region("europe-west1")
+            .firestore.document("/animals/{animalName}")
+            .onDelete(async (snapshot, context) => {
+                receivedSnapshots.push(snapshot)
+            })
+
+        // When a doc is created in that collection;
+        await driver
+            .firestore()
+            .collection("animals")
+            .doc("tiger")
+            .set({
+                colour: "orange", size: "large",
+            })
+
+        // And Firebase finishes its jobs;
+        await driver.jobsComplete()
+
+        // Then the handler should not be triggered.
+        expect(receivedSnapshots).toHaveLength(0)
+    })
+
+    test("onDelete handler is triggered on doc delete", async () => {
+        // Given we set up an onDelete handler on a collection;
+        const receivedSnapshots: IFirestoreDocumentSnapshot[] = []
+        driver
+            .runWith()
+            .region("europe-west1")
+            .firestore.document("/animals/{animalName}")
+            .onDelete(async (snapshot, context) => {
+                receivedSnapshots.push(snapshot)
+            })
+
+        // When a doc is created in that collection;
+        await driver
+            .firestore()
+            .collection("animals")
+            .doc("tiger")
+            .set({ colour: "orange", size: "large" })
```

Comment:
Have you run a formatter over this? On line 27 we have exactly the same thing but it's split over 3 lines